app/src/main/java/com/tvtracker/api/ImdbInterceptor.kt

```diff
@@ -9,7 +9,7 @@ class ImdbInterceptor: Interceptor {
     override fun intercept(chain: Interceptor.Chain): Response {
         val request = chain.request()
             .newBuilder()
-            .addHeader("x-rapidapi-key", BuildConfig.xrapidapikey)
+//          .addHeader("x-rapidapi-key", xrapidapikey)
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.